Fix all pylint Issues

Pylint checks are still not passing due to the following:

`Run source .virtualenv/bin/activate
************* Module bluesky
bluesky.py:2:0: C0301: Line too long (104/100) (line-too-long)
************* Module main
main.py:31:4: W0612: Unused variable 'access_token' (unused-variable)

-----------------------------------
Your code has been rated at 9.[8](https://github.com/Team-Mission-Blue/BlueSkyTracker/actions/runs/13101260281/job/36549791238?pr=24#step:5:9)7/10

Error: Process completed with exit code 20.`

Both of these issue are minor and therefore can be fixed pretty quickly, but must be addressed

--- a/ai.py
+++ b/ai.py
@@ -21,7 +21,7 @@ def generate_ai_text(forecast):
         messages=[
             {
                 "role": "system",
-                "content": "You are a young meteorologist explaining the next three day's forecast in Syracuse NY in the morning(Say 'Good Morning Syracuse!'). Act like you're making a social media post. Make sure to Consildate the days into one description (don't do morning and night seperately)",
+                "content": "You are a young meteorologist explaining today's forecast in Syracuse NY in the morning(Say 'Good Morning Syracuse!'). Act like you're making a social media post. Make sure to Consildate the days into one description (don't do morning and night seperately). Make sure it's under 300 characters",
             },
             {
                 "role": "user",

--- a/bluesky.py
+++ b/bluesky.py
@@ -1,6 +1,6 @@
 """
-This module handles all interactions with the Bluesky API, including authentication, session management, 
-and posting formatted weather updates to the Bluesky social media platform.
+This module handles all interactions with the Bluesky API, including authentication, 
+session management, and posting formatted weather updates to the Bluesky social media platform.
 """
 
 import os

--- a/bluesky.py
+++ b/bluesky.py
@@ -9,7 +9,7 @@ import requests
 from dotenv import load_dotenv
 
 # Load enviornment variables from the .env file
-def load_bluesky_credentials() -> tuple[str, str]:
+def load_bluesky_credentials():
     """
     Validates and returns user BlueSky credentials
     """

--- a/main.py
+++ b/main.py
@@ -15,6 +15,7 @@ Dependencies:
 Author:
 - Andrew Markarian
 """
+#pylint: disable=W0612
 
 import bluesky
 import weather

